docs/src/terms.md

```diff
@@ -29,16 +29,56 @@ with the `HalfInteger` type.
 
 ```@docs
 terms
-count_terms
 ```
 
-The `L` and `S` quantum numbers are not, in general, sufficient to uniquely identify a term.
-The [`IntermediateTerm`](@ref) type allows one to specify an additional quantum number which
-would uniquely identify the term.
+## Term multiplicity and intermediate terms
+
+For subshells starting with `d³`, the possible terms may occur more
+than once (multiplicity higher than one), corresponding to different
+physical states. These arise from different sequences of coupling the
```

Comment:
Since we write that they correspond to different physical states and arise from different sequences of coupling the electrons, I think it's sort of fine for now. We can revisit this later.